raiden/utils/typing.py

```diff
@@ -57,6 +57,8 @@
 T_MessageID = int
 MessageID = NewType('MessageID', T_MessageID)
 
+UDPMessageID = Tuple[str, int, Address]
```

Comment:
note that there is going to be a part in the code where type would be a union between `MessageID` and `UDPMessageID` but we can't do anything to avoid it.

Reply:
I'm curious.

Reply:
I will see when it happens.